README.md

```diff
@@ -6,25 +6,37 @@ Local coordinator for offline development of Hathora applications.
 
 ## How to use
 
-### Setup
+### 1. Enable HTTPS for localhost
 
-Generate certs using mkcert:
+First, generate certificates using [mkcert](https://github.com/FiloSottile/mkcert).
+
+In your project root directory:
 
 ```sh
 mkcert -install
 mkcert localhost
 ```
 
-Install dependencies:
+Make sure to add `*.pem` to your `.gitignore` file to avoid committing the certificate files.
```

Comment:
is this needed? *.pem is already in the gitignore for this repo

Reply:
oh i see you're suggesting installing the local-coordinator into your hathora project